components/brave_vpn/brave_vpn_os_connection_api.cc

```diff
@@ -250,21 +239,20 @@ void BraveVPNOSConnectionAPI::UpdateAndNotifyConnectionStateChange(
   if (connection_state_ == state)
     return;
 
-#if BUILDFLAG(IS_WIN)
-  // On Windows, we get disconnected status update twice.
-  // When user connects to different region while connected,
-  // we disconnect current connection and connect to newly selected
-  // region. To do that we monitor |DISCONNECTED| state and start
-  // connect when we get that state. But, Windows sends disconnected state
-  // noti again. So, ignore second one.
-  // On exception - we allow from connecting to disconnected in canceling
-  // scenario.
+  // Ignore disconnected state while connecting is in-progress.
```

Comment:
These updates to comments lose the context for WHEN this can happen- which is when person is changing regions. 

Can we add that back in?
```
// When user connects to different region while connected,
// we disconnect current connection and connect to newly selected
// region.
```

Reply:
wait- maybe I missed the actual change 😅 You're correct to delete this, sorry!